docusaurus/docs/reference/metadata.md

```diff
@@ -65,12 +65,12 @@ Plugin dependency. Used to display information about plugin dependencies in the
 
 #### Properties
 
-| Property  | Type   | Required | Description                                                                                                                                   |
-| --------- | ------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
-| `id`      | string | **Yes**  |                                                                                                                                               |
-| `name`    | string | **Yes**  |                                                                                                                                               |
-| `type`    | string | **Yes**  | Possible values are: `app`, `datasource`, `panel`.                                                                                            |
-| `version` | string | **Yes**  | We suggest using [SemVer](https://semver.org/) for your plugin versioning to alert users about potential changes, new features, or bug fixes. |
+| Property  | Type   | Required | Description                                                                                                                                           |
+| --------- | ------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`      | string | **Yes**  |                                                                                                                                                       |
+| `name`    | string | **Yes**  |                                                                                                                                                       |
+| `type`    | string | **Yes**  | Possible values are: `app`, `datasource`, `panel`.                                                                                                    |
+| `version` | string | **No**   | We suggest using [SemVer](https://semver.org/) as your plugin versioning scheme. This property should use a specific, pinned version and not a range. |
```

Comment:
Maybe add something along the lines "if no version specified, the latest supported version will be used"?

Reply:
Ah thanks @marefr . Added in c2858fbd4dbc0a8324b0ace4a871afe613081308 👍

Reply:
It's not really up to the developer either as I'm pretty sure all our tooling and some checks within codebases is geared around Semver and if they were to use anything else (such as date of release) things would go wrong somewhere. How about I reword it to:

```
This property should use a specific, pinned [SemVer](https://semver.org/) version and not a range. If no version is specified, it will resolve to the latest supported version.
```

and we update the `info.version` property blurb to read something like:

```diff
-Project version of this commit, e.g. `6.7.x`
+[SemVer](https://semver.org/) version of the plugin, e.g. `6.7.1`.
```

Reply:
LGTM 🚀